### PR TITLE
support pause all audio while call game.pause

### PR DESCRIPTION
**Pull request by bofeng-song**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/audio/audio-manager.ts [hunk 1]
```diff
@@ -102,6 +102,24 @@ export class AudioManager {
...
         }
     }
+
+    public pause () {
+        this._oneShotAudioInfoList.forEach((info) => {
+            info.audio.stop();
+        });
+        this._audioPlayerInfoList.forEach((info) => {
+            // eslint-disable-next-line no-void
+            void info.audio.pause();
```
Comment (minggo) [inline cocos/audio/audio-manager.ts]: Why need `void`?

Comment (bofeng-song) [reply cocos/audio/audio-manager.ts]: Promises must be handled appropriately or explicitly marked as ignored with the `void` operator.

Comment (PPpro) [inline cocos/audio/audio-manager.ts]:
```suggestion
            info.audio.pause().catch(e => {});
```

File: cocos/audio/audio-manager.ts [hunk 2]
```diff
@@ -102,6 +102,24 @@ export class AudioManager {
...
+        });
+    }
+
+    public resume () {
+        // onShotAudio can not be resumed
+        this._audioPlayerInfoList.forEach((info) => {
+            // eslint-disable-next-line no-void
+            void info.audio.play();
```
Comment (PPpro) [inline cocos/audio/audio-manager.ts]: the same

**Review by minggo — APPROVED** [review]

**Review by PPpro — APPROVED** [review]

**Pull request merged**